File: src/integration-tests/expecteds/py/multi-schema.py
from typing import NewType
from typing import Union
from typing import List
from typing import Any

StringDoaGddGA = NewType("StringDoaGddGA", str)

OneOfStringDoaGddGAStringDoaGddGABERs71N5 = NewType("OneOfStringDoaGddGAStringDoaGddGABERs71N5", Union[StringDoaGddGA, StringDoaGddGA])
"""A way to describe color
"""
RGBOrHex = NewType("RGBOrHex", List[OneOfStringDoaGddGAStringDoaGddGABERs71N5])
"""Human word for color
"""
StringDBm1TnLT = NewType("StringDBm1TnLT", str)

ColorAsAnything = NewType("ColorAsAnything", Any)
"""Generated! Represents an alias to any of the provided schemas
"""
AnyOfRGBOrHexStringDBm1TnLTColorAsAnything = NewType("AnyOfRGBOrHexStringDBm1TnLTColorAsAnything", Union[RGBOrHex, StringDBm1TnLT, ColorAsAnything])

File: src/integration-tests/expecteds/rs/multi-schema.rs
extern crate serde;
extern crate serde_json;
extern crate derive_builder;

use serde::{Serialize, Deserialize};
pub type StringDoaGddGA = String;
#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
#[serde(untagged)]
pub enum OneOfStringDoaGddGAStringDoaGddGABERs71N5 {
    StringDoaGddGA(StringDoaGddGA),
    StringDoaGddGA(StringDoaGddGA),
}
/// RGBOrHex
///
/// A way to describe color
///
pub type RGBOrHex = Vec<OneOfStringDoaGddGAStringDoaGddGABERs71N5>;
/// StringDBm1TnLT
///
/// Human word for color
///
pub type StringDBm1TnLT = String;
pub type ColorAsAnything = serde_json::Value;
#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
#[serde(untagged)]
pub enum AnyOfRGBOrHexStringDBm1TnLTColorAsAnything {
    RGBOrHex(RGBOrHex),
    StringDBm1TnLT(StringDBm1TnLT),
    ColorAsAnything(ColorAsAnything),
}
